docs/geedocs/5.2.2/index.html

```diff
@@ -40,7 +40,7 @@ <h4><a href="answer/2987876.html">Legal notices</a></h4>
   </div>
   <div class="footer">
     <hr />
-    <p class="copyright">&copy;2017 Google</p>
+    <p class="copyright">&copy;2017 Google, &copy;2018 Open GEE Contributors</p>
```

Comment:
2017-2018 Open GEE Contributors ?. Since this document was edited in 2017 as well.